### DISCUSSION TITLE
Configure Deye SUN600G3-EU-230

### DISCUSSION THREAD

**Discussion by Panoramiac** [StephanJoubert/home_assistant_solarman#127] [Q&A] [answered]
I try to configure my new Deye SUN600G3-EU-230 Inverter. I installed it in my latest Home Assistant using Hacs. Port 8899 is open. The serial starts with 41xxxxxxxx. I can not get any Information from the Inverter. In the Solarman App it works. I tried the hybrid and the serial configuration so far.

**Comment by Panoramiac** [comment, marked as answer]
It seems to work now. I am using this configuration:

![Screenshot_20221031-093640_Home Assistant](https://user-images.githubusercontent.com/48438802/198966150-e2a04181-7f30-4265-98a7-13fe0a083d87.jpg)

**Reply by Panoramiac** [reply]
I configured now the current production, which is in W, not in KWh:
![image](https://user-images.githubusercontent.com/48438802/198973696-d048b0b5-b5e3-4de3-a8c4-50937e33bd44.png)

**Reply by optimismus** [reply]
I just opened an issue for this (W not kwh). Could you give me an hint where to find the files to correct the entity?
Regards
Axel

**Reply by hajo62** [reply]
Look in this folder: https://github.com/StephanJoubert/home_assistant_solarman/tree/main/custom_components/solarman/inverter_definitions

**Reply by optimismus** [reply]
@hajo62 Sorry for beeing unclear. I tried to find the file on my HASS with filemanager, but can't find it.

**Reply by hajo62** [reply]
I'm running HA on docker. There it is \<homeassistant-home-folder\>/custom_components/solarman/inverter_definitions
Maybe that helps.

**Reply by optimismus** [reply]
@hajo62 Thank you, that was it!

**Comment by hajo62** [comment]
Having a look at the code ([deye_string.yaml](https://github.com/StephanJoubert/home_assistant_solarman/blob/main/custom_components/solarman/inverter_definitions/deye_string.yaml)) , that seems to be corrected 2 month ago.
But the release is from April.

How can we bring this into an actualized version?

**Reply by Panoramiac** [reply]
I asked this also already, also on discord.

**Reply by hajo62** [reply]
And if brought to a new version, maybe the temperature could be added, too?
See https://github.com/StephanJoubert/home_assistant_solarman/discussions/81#discussioncomment-3416294. Best would be if the 10 °C offset are directly subtracted 😀

**Comment by mirkoebert** [comment]
> I try to configure my new Deye SUN600G3-EU-230 Inverter. I installed it in my latest Home Assistant using Hacs. Port 8899 is open. The serial starts with 41xxxxxxxx. I can not get any Information from the Inverter. In the Solarman App it works. I tried the hybrid and the serial configuration so far.

I found in my log following line:
2023-02-13 21:57:27.251 WARNING (SyncWorker_4) [custom_components.solarman.solarman] Querying inverter 3901XXXXXX at 192.168.1.156:8899 failed on connection start with exception [OSError]

It looks that HA can't reach the inverter:
2023-02-15 12:17:16.974 WARNING (MainThread) [homeassistant.helpers.entity] Update of sensor.solarman_pv1_power is taking over 10 seconds
2023-02-15 12:17:22.783 WARNING (SyncWorker_2) [custom_components.solarman.solarman] Querying registers [150 - 248] failed, aborting.
